tests/generation/test_utils.py

```diff
@@ -1570,7 +1567,8 @@ def test_past_key_values_format(self):
                     )
 
     @pytest.mark.generate
-    def test_generate_from_inputs_embeds_decoder_only(self):
+    def test_generate_from_inputs_embeds(self):
+        """Tests that we can generate from `inputs_embeds` instead of `input_ids` in LLMs, VLMs, etc"""
```

Comment:
Related to the deleted `inputs_embeds` tests -- clarifies that this test also runs for VLMs